### ISSUE TITLE
Calendar does not show correct date order

### ISSUE THREAD

**Issue by MakuSensei** [Drutol/MALClient#149]
Hello, I want to report a problem with the calendar. The problem is that the dates are incorrect and show an anime on another day that does not match.
For example an anime that is broadcast on Fridays is shown in the column on Saturdays. 
Nor can I correct it in settings and calendar. Be putting one day less, it ends up changing everything and it does not look good. 
My recommendation is to allow the time zone change. For example in my case -3 because I am from Argentina.
In short this myanimelist client is almost perfect. You need to improve the calendar so you do not need to use livechart for such use.
Sorry if you do not understand correctly what I write, I use the Google translator android translator.
Greetings from Argentina

_Sent from my Xiaomi Redmi 3S using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Drutol** [comment]
Hello, yeah I noticed it and thanks to yesterdays update I'll be able to fix it ^^. Right now values are taken from MAL's "broadcasting" info which is in JST and then I offset this to device timezone. I got my hands yesterday on great data provided by iMAL dev with exact airing times so next update should greatly improve countdowns. :)

**Comment by MakuSensei** [comment]
Thanks for the update, now looks better. 
Only detail is that it does not show in the Tuesday column the anime NEW GAME !!

_Sent from my Xiaomi Redmi 3S using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Drutol** [comment]
Got it, fixed. I'll push hotfix in a while.

**Comment by Drutol** [comment]
Hotfix up

**Comment by MakuSensei** [comment]
Hello the calendar is still not showing the anime NEW GAME

**Comment by Drutol** [comment]
Are you on newest version?

**Comment by MakuSensei** [comment]
Yes I am in the latest version

**Comment by fatflyingpigs** [comment]
I can confirm that using the version that was updated on Aug 28th, _New Game!!_ does not show on the calendar. These other airing anime do not show up either: _Nana Maru San Batsu_, _Isekai wa Smartphone to Tomo ni._ and _Youkoso Jitsuryoku Shijou Shugi no Kyoushitsu e (TV)_.

Since you mentioned the iMAL dev in an above comment, I can confirm that iMAL _does_ show the dates for the shows I listed. Hopefully this helps some, big fan of your app!

**Comment by Drutol** [comment]
![screenshot_2017-08-31-09-09-07-719_com drutol malclient](https://user-images.githubusercontent.com/6102530/29910924-38f565be-8e2c-11e7-804a-ca7366f9d6ff.png)

**Comment by Drutol** [comment]
Well, that's strange. Did you check calendar settings? Maybe it's in your ptw...

**Comment by Drutol** [comment]
Also the calendar was limited to 20 shows due to old data gathering mechanism, I've upped it to 40 in v1.2.4.0.

**Comment by MakuSensei** [comment]
thanks in this latest update already appears the best
